tests/model_hub_tests/models_hub_common/test_performance_model.py

```diff
@@ -0,0 +1,220 @@
+            inputs_info, timedelta = wrap_timer(self.get_inputs_info, (model_obj,))
+            print('Got inputs info in {} secs'.format(timedelta))
+            print("Prepare input data")
+            results.status = Status.PREPARE_INPUTS
+            inputs = self.prepare_inputs(inputs_info)
+            print("Convert the model into ov::Model")
+            results.status = Status.GET_CONVERTED_MODEL
+            converted_model = self.compile_model(self.get_converted_model(model_obj), ie_device)
+            print("read the model into ov::Model")
+            results.status = Status.GET_READ_MODEL
+            read_model = self.compile_model(self.get_read_model(model_obj), ie_device)
+            print("Infer the converted model")
+            results.status = Status.INFER_CONVERTED_MODEL
+            converted_model_time, converted_model_time_variance = self.infer_model(converted_model, inputs)
+            print('converted model time infer {}'.format(converted_model_time))
+            print('converted model time infer var {}'.format(converted_model_time_variance))
+            print("Infer read model")
+            results.status = Status.INFER_READ_MODEL
+            read_model_time, read_model_time_variance = self.infer_model(read_model, inputs)
+            print('read model time infer {}'.format(read_model_time))
+            print('read model time infer var {}'.format(read_model_time_variance))
+
+            infer_time_ratio = converted_model_time/read_model_time
+
+            results.converted_infer_time = converted_model_time
+            results.converted_model_time_variance = converted_model_time_variance
+            results.read_model_infer_time = read_model_time
+            results.read_model_infer_time_variance = read_model_time_variance
+            results.infer_time_ratio = infer_time_ratio
+
+            if abs(infer_time_ratio - 1) > TestModelPerformance.threshold_ratio:
+                if (read_model_time_variance > TestModelPerformance.threshold_var
+                        or converted_model_time_variance > TestModelPerformance.threshold_var):
+                    results.status = Status.LARGE_INFER_TIME_DIFF_WITH_LARGE_VAR
+                    results.error_message = "too large ratio {} with large variance".format(infer_time_ratio)
+                else:
+                    results.status = Status.LARGE_INFER_TIME_DIFF
+                    results.error_message = "too large ratio {}".format(infer_time_ratio)
+            else:
+                results.status = Status.OK
+        except:
+            ex_type, ex_value, tb = sys.exc_info()
+            results.error_message = "{tb}\n{ex_type}: {ex_value}".format(tb=''.join(traceback.format_tb(tb)),
+                                                             ex_type=ex_type.__name__, ex_value=ex_value)
+        return results
+
+    def run(self, model_name, model_link, ie_device):
+        self.result = Results()
+        t0 = time.time()
```

Comment:
I think we do need to measure time at this level.

Reply:
why not? it's not a problem to delete it, if it will be useless

Reply:
Ok. agree. Not critical